x-pack/plugins/licensing/common/license_update.ts

```diff
@@ -18,7 +27,10 @@ export function createLicenseUpdate(
   initialValues?: ILicense
 ) {
   const manuallyRefresh$ = new Subject<void>();
-  const fetched$ = merge(triggerRefresh$, manuallyRefresh$).pipe(exhaustMap(fetcher), share());
+  const fetched$ = merge(
+    triggerRefresh$.pipe(takeUntil(stop$.pipe(tap(() => manuallyRefresh$.complete())))),
+    manuallyRefresh$
+  ).pipe(exhaustMap(fetcher), share());
```

Comment:
What I've noticed is that `$triggerRefresh` remains subscribed to, and provided that it's an RxJS timer, it continues to emit values forever. So the goal is not to prevent `fetcher` from being called, but rather to properly stop the `timer()`.

> Also: do we still need to pipe cache$ though takeUntil(stop$) with that change?

You're right, we can remove it from the `cache$` operator.

Reply:
I've seen there's a little mess with the deprecated `publishReplay` in this logic, perhaps I can try to simplify / update it a bit.